src/components/Card/Card.test.tsx

```diff
@@ -1,37 +1,98 @@
-import { render } from '@testing-library/react';
 import React from 'react';
+import { render, screen } from '@testing-library/react';
 
 import { Card, CardProps } from './';
 
+import styles from './Card.module.scss';
+
 const DEFAULT_PROPS: CardProps = {
-  title: 'Title Text',
-  value: '123',
-  bottomText: 'Bottom Text'
+  title: '',
+  value: '',
+  bottomText: undefined
 };
 
+const MOCK_SKELETON_ID = 'mock-skeleton';
+
 jest.mock('../Skeleton', () => ({
-  Skeleton: () => <span>Skeleton</span>
+  Skeleton: () => <div data-testid={MOCK_SKELETON_ID} />
 }));
 
-const renderComponent = (props?: CardProps) => render(<Card {...props} />);
+describe('<Card />', () => {
+  test('should only render text in the correct order - title, value, bottom text', () => {
```

Comment:
The way I read this test suggests a specific order of ```1,2,3```, _only_. Might be a little overkill(?), but should checks be added to confirm text content won't be in an unexpected order, something like ```expect(container).not.toHaveTextContent('3,2,1')```? 

Although, I guess this is part of the reason you've added commas to ensure an exact match because without them something like ```expect(container).not.toHaveTextContent('321')``` should still pass.

Reply:
This test here is here to assert that the visible elements are rendered in the order `title`, `value`, `bottom text`. I think my use of `only` might have made this test a bit confusing, but in essence it's checking that we're seeing:
```
_Title_
**Value**
_Bottom Text_
```
Rather than some other variation of the above, e.g:
```
**Value**
_Title_
_Bottom Text_
```
My thinking is that we can assert that this is true by using three arbitrary strings as props and checking that the text content of the container is equal to those three strings concatenated in the right order. If we had `title=1`, `value=2`, and `bottomText=3`, and the container text content was `123`, we would know that the text ordering is correct.

I put commas in each of the strings just so the test output reads a little nicer - they're not important to the test. Perhaps this was a little bit confusing and should be changed.

I think you're right that I need to use something stricter than `toHaveTextContent` though as it's not an exact match. I will change this to `expect(container.textContent).toBe('1,2,3')`.

Reply:
Yeah nice I like this - ```expect(container.textContent).toBe('1,2,3')```. I copied the above for ordering of TabsNav but I'll go back and change it to this, makes sense